modules/core/arrow-test/src/main/kotlin/arrow/test/UnitSpec.kt

```diff
@@ -11,28 +10,19 @@ import io.kotlintest.specs.AbstractStringSpec
  */
 abstract class UnitSpec : AbstractStringSpec() {
 
+  private val lawTestCases = mutableListOf<TestCase>()
+
   fun testLaws(vararg laws: List<Law>): List<TestCase> {
```

Comment:
Can this be an expression body like

`fun testLaws(vararg laws: List<Law>): List<TestCase> = laws.flatMap { ...`